docs/core/tools/dotnet.md

```diff
@@ -208,6 +208,17 @@ Command | Function
 [dotnet nuget remove source](dotnet-nuget-remove-source.md) | Removes a NuGet source.
 [dotnet nuget update source](dotnet-nuget-update-source.md) | Updates a NuGet source.
 
+### Workload commands
+
+Command | Function
+--- | ---
+[dotnet workload install](dotnet-workload-install.md) | Installs an optional workload.
+[dotnet workload list](dotnet-workload-list.md) | Lists all installed workloads.
+[dotnet workload repair](dotnet-workload-repair.md) | Repairs all installed workloads.
+[dotnet workload search](dotnet-workload-search.md) | List selected workloads or all available workloads.
+[dotnet workload uninstall](dotnet-workload-install.md) | Uninstalls a workload.
+[dotnet workload update](dotnet-workload-update.md) | Reinstalls all installed workloads.
```

Comment:
@wli3 is also adding a restore command that we'll want to include, although we can add that in a separate PR.